config/staging.yml.erb

```diff
@@ -13,3 +13,6 @@ custom_error_response: false
 
 # Configurations for making calls to the Immersive Reader API on Azure
 imm_reader_client_id:     '74937af0-55ee-411a-867a-57eefadec7d9'
+
+# General CDO Amplitude key
+cdo_amplitude_api_key: !Secret
```

Comment:
I think this is fine but I'm confused still why staging broke before. It's definitely possible y'all understand and I'm still playing catchup so if that's the case, feel free to merge :)

Reply:
I definitely don't understand why it broke, so if you have any other ideas we should try those too!